guide/ops/starting-stopping-monitoring.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Starting, Stopping and Monitoring
+layout: website-normal
+---
+
+**NOTE:** This document is for information on starting an Apache Brooklyn
+Server.  For information on using the Brooklyn Client CLI to access an already
+running Brooklyn Server, refer to [Client CLI Reference](cli/index.html).
+
+
+## Starting
+
+To launch Brooklyn, from the directory where Brooklyn is unpacked, run:
+
+{% highlight bash %}
+% bin/brooklyn launch > /dev/null 2>&1 & disown
```

Comment:
@aledsage My opinion is that this page serves a different purpose to the CLI reference page and we should keep both. This is duplicating a just a few lines, and then sending the reader to the Server CLI page. I think this is the smallest possible amount of duplication that keeps both pages useful.